tools/extract_particle.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
```

Comment:
Does it work with python3?

Reply:
No.

Reply:
We need to make sure all our scripts are compatible with Python3.

Reply:
https://pythonclock.org/, you have 1 year 1 month ;-)